test/validation_test.go

```diff
@@ -56,6 +58,114 @@ func kustomizeBuild(dir string) ([]byte, []byte, error) {
 	return outBuf.Bytes(), errBuf.Bytes(), err
 }
 
+func testNamespaceResources(t *testing.T) {
+	t.Parallel()
+
+	// All namespaces defined in neco-apps should have the `team` label.
+	// Exceptionally, `sandbox` ns should not have the `team` label.
+	doCheckKustomizedYaml(t, func(t *testing.T, data []byte) {
+		var meta struct {
+			metav1.TypeMeta   `json:",inline"`
+			metav1.ObjectMeta `json:"metadata,omitempty"`
+		}
+		err := yaml.Unmarshal(data, &meta)
+		if err != nil {
+			t.Fatal(err)
+		}
+		if meta.Kind != "Namespace" {
+			return
+		}
+
+		// `sandbox` namespace should not have a team label.
+		if meta.Name == "sandbox" {
+			if meta.Labels["team"] != "" {
```

Comment:
```suggestion
			if _, ok := meta.Labels["team"]; ok {
```

My modified check rejects the label of `team: ""`.
Though this type of label may cause no harm, it heavily depends on the context.
So I prefer strict check.